src/css/schema/migrate-None-to-0.sql

```diff
@@ -0,0 +1,39 @@
+-- MySQL Script generated by MySQL Workbench
+-- Thu Jul 30 11:58:38 2015
+-- Model: New Model    Version: 1.0
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
```

Comment:
We should probably get rid of these lines and the comment above that  "MySQL Script generated by MySQL Workbench", etc.